{"created": 1756806123.8527553, "duration": 0.18109345436096191, "exitcode": 2, "root": "/home/jayasree/e-commerce/full-stack-fastapi-template/backend", "environment": {}, "summary": {"total": 0, "collected": 20}, "collectors": [{"nodeid": "", "outcome": "passed", "result": [{"nodeid": "test_passed1.py", "type": "Module"}, {"nodeid": "test_failed1.py", "type": "Module"}]}, {"nodeid": "test_passed1.py", "outcome": "failed", "result": [], "longrepr": "../../venv/lib/python3.12/site-packages/_pytest/python.py:617: in _importtestmodule\n    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)\n../../venv/lib/python3.12/site-packages/_pytest/pathlib.py:567: in import_path\n    importlib.import_module(module_name)\n/usr/lib/python3.12/importlib/__init__.py:90: in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\n<frozen importlib._bootstrap>:1387: in _gcd_import\n    ???\n<frozen importlib._bootstrap>:1360: in _find_and_load\n    ???\n<frozen importlib._bootstrap>:1331: in _find_and_load_unlocked\n    ???\n<frozen importlib._bootstrap>:935: in _load_unlocked\n    ???\n../../venv/lib/python3.12/site-packages/_pytest/assertion/rewrite.py:177: in exec_module\n    source_stat, co = _rewrite_test(fn, self.config)\n../../venv/lib/python3.12/site-packages/_pytest/assertion/rewrite.py:359: in _rewrite_test\n    tree = ast.parse(source, filename=strfn)\n/usr/lib/python3.12/ast.py:52: in parse\n    return compile(source, filename, mode, flags,\nE     File \"/home/jayasree/e-commerce/full-stack-fastapi-template/backend/test_passed1.py\", line 427\nE       assert result == created_userimport pytest\nE                                           ^^^^^^\nE   SyntaxError: invalid syntax"}, {"nodeid": "test_failed1.py", "outcome": "passed", "result": [{"nodeid": "test_failed1.py::test_send_email_1_positive", "type": "Function", "lineno": 141}, {"nodeid": "test_failed1.py::test_send_email_2_negative_invalid_email_to", "type": "Function", "lineno": 166}, {"nodeid": "test_failed1.py::test_send_email_3_edge_empty_content", "type": "Function", "lineno": 191}, {"nodeid": "test_failed1.py::test_send_email_4_negative_emails_disabled", "type": "Function", "lineno": 213}, {"nodeid": "test_failed1.py::test_send_email_5_edge_only_email_to", "type": "Function", "lineno": 234}, {"nodeid": "test_failed1.py::test_send_email_6_negative_message_send_raises_exception", "type": "Function", "lineno": 256}, {"nodeid": "test_failed1.py::test_send_email_7_edge_smtp_host_none", "type": "Function", "lineno": 281}, {"nodeid": "test_failed1.py::test_send_email_8_negative_emails_from_email_none", "type": "Function", "lineno": 305}, {"nodeid": "test_failed1.py::test_send_email_9_edge_smtp_port_zero", "type": "Function", "lineno": 329}, {"nodeid": "test_failed1.py::test_send_email_10_negative_smtp_auth_fails", "type": "Function", "lineno": 354}, {"nodeid": "test_failed1.py::test_create_user_1_positive", "type": "Function", "lineno": 386}, {"nodeid": "test_failed1.py::test_create_user_2_negative_email_exists", "type": "Function", "lineno": 413}, {"nodeid": "test_failed1.py::test_create_user_3_edge_empty_email", "type": "Function", "lineno": 440}, {"nodeid": "test_failed1.py::test_create_user_4_negative_crud_create_user_raises_exception", "type": "Function", "lineno": 463}, {"nodeid": "test_failed1.py::test_create_user_5_edge_very_long_email", "type": "Function", "lineno": 487}, {"nodeid": "test_failed1.py::test_create_user_6_negative_emails_disabled", "type": "Function", "lineno": 515}, {"nodeid": "test_failed1.py::test_create_user_7_edge_crud_get_user_by_email_raises_exception", "type": "Function", "lineno": 538}, {"nodeid": "test_failed1.py::test_create_user_8_negative_generate_new_account_email_raises_exception", "type": "Function", "lineno": 562}, {"nodeid": "test_failed1.py::test_create_user_9_edge_user_in_password_none", "type": "Function", "lineno": 588}, {"nodeid": "test_failed1.py::test_create_user_10_negative_user_in_email_none", "type": "Function", "lineno": 617}]}], "tests": []}